src/mscorlib/src/System/Security/SecurityException.cs

```diff
@@ -668,16 +668,16 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
             // Stubs for surface area compatibility only
             public SecurityException(string message, System.Type type) { }
```

Comment:
The constructor should call base class constructor and set the properties, something like:

```
    public SecurityException(String message, Type type) 
            : base(message)
    {
        SetErrorCode(System.__HResults.COR_E_SECURITY);
        PermissionType = type;
    }

    public SecurityException(string message, System.Type type, string state)
        : base(message)
    {
        SetErrorCode(System.__HResults.COR_E_SECURITY);
        PermissionType = type;
        PermissionState = state;
    }
```